tests/system_tests_http2.py

```diff
@@ -297,15 +296,16 @@ def check_connector_delete(self, client_addr, server_addr):
                 break
         self.assertFalse(http_server_conn_found)
 
-        sleep(2)
-
         # Now, run a curl client GET request with a timeout
         request_timed_out = False
-        try:
-            out = self.run_curl(client_addr, timeout=5)
-            print(out)
-        except Exception as e:
-            request_timed_out = True
+        for n in range(3):
```

Comment:
looks good, but I think you should first merge https://github.com/skupperproject/skupper-router/pull/101 and then rebase the pr, just to be sure